build/config/BUILDCONFIG.gn

```diff
@@ -535,7 +535,7 @@ if (custom_toolchain != "") {
 #
 # Variables
 #   no_default_deps: If true, no standard dependencies will be added.
-if (is_android || (is_linux && current_cpu != "x86")) {
+if (!is_win && current_cpu != "x86") {
```

Comment:
This previously would pass for android of non x86, now it doesn't.  Was that intended?

Reply:
No. Fixing.